pom.xml

```diff
@@ -132,17 +132,20 @@
     <jaxb.version>2.3.0</jaxb.version>
     <jersey.version>2.27</jersey.version>
     <jetty.version>9.4.17.v20190418</jetty.version>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.release>11</maven.compiler.release>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <!-- surefire/failsafe plugin option -->
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
+    <!-- temporary until other Java 11 issues are resolved -->
+    <modernizer.skip>true</modernizer.skip>
```

Comment:
Would it make sense to create a 2.1 issue for this?

Reply:
I've already got another PR ready to go for those items, that I'm working on, that will be uploaded shortly after this.